keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,440 @@
+code freeze remains in effect until the last week of the release, so 
+development for the next release generally starts prior to the official release 
+team kickoff. A minimum of 1 week is needed between releases to fully form the 
+release team and to facilitate on-boarding of shadows. The fourth release of 
+the year has traditionally been compressed and limited in scope, overlapping 
+with end of year holidays and vacation for many contributors. Additionally, 
+KubeCon normally occurs during at least one release, eliminating at *least* one week of 
+working time.
+
+*Kubernetes Release Sechedule 2021 (Existing 4 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 2     | 1     | 1 (January 11) | |
+| 14 | 1 | 13 (April 8) | |
+| 16 | 2 | 1 (April 19) | |
+| 27 | 2 | 11 (July 06) | One week break for KubeCon EU | 
+| 29 | 3 | 1 (July 20) | |
+| 40 | 3 | 11 (October 5) | | 
+| 42 | 4 | 1 (October 18) | |
+| 52 | 4 | 10 (December 28) | End of Year Holidays |
+
+*Kubernetes Release Sechedule 2022 (Existing 4 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 3) | |
+| 12 | 1 | 12 (March 15) | |
+| 14 | 2 | 1 (March 28) | Probable KubeCon EU Break|
+| 26 | 2 | 12 (June 28) | |
+| 28 | 3 | 1 (July 11) | | 
+| 40 | 3 | 12 (October 4) | | 
+| 42 | 4 | 1 (October 17) | Probably KubeCon NA Break | 
+| 52 | 4 | 10 (Dec 28) | | 
+
+With the proposed change in cadence, the notional schedules for the remainder of
+2021 and 2022 are shown below:
+
+*Kubernetes Release Schedule 2021 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 2  | 1 | 1 (January 11) | |
+| 14 | 1 | 13 (April 8) | |
+| 17 | 2 | 1 (April 26) | |
+| 32 | 2 | 15 (August 02) | KubeCon EU Break (May 4-7) |
+| 35 | 3 | 1 (August 30) | | 
+| 50 | 3 | 15 (December 14) | Kubecon Break (Oct 12-15) | 
+
+*Kubernetes Release Sechedule 2022 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 3) | |
+| 15 | 1 | 15 (April 12) | | 
+| 18 | 2 | 1 (May 2) | Probably KubeCon EU |
+| 33 | 2 | 15 (August 15) | |
+| 36 | 3 | 1 (September 6 | Probably KubeCon NA | 
+| 51 | 3 | 15 (December 20) | 
+
+
+This KEP will be in the `alpha` stage for the Kubernetes 1.22 Release. During 
+this time, SIG-Release will focus on communication of the cadence change through
+all available mechanisms. The KEP will promote to the `beta` stage for the 
+Kubernetes 1.23 Release, which will be the final release of 2021. After 
+the 1.23, 1.24, and  1.25 Releases, will will collect feedback and incorporate 
+that feedback into the lightweight framework surrounding release schedule 
+development and promote this KEP to `stable` for the 1.26 Release. 
```

Comment:
let's add a summary table like this? (after the paragraph above ... not instead, i like the commentary, we should keep it)

```
1.22 - Alpha 
1.23/1.24/1.25 - Beta
1.26 - Stable
```

Reply:
added!

Reply:
thanks!